books/free-programming-books-bl.md

```diff
@@ -142,6 +143,14 @@
 * [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com) - Devsonket Team
 
 
+### MySQL
+
+* [Bangla MySQL Database Tutorials](https://www.youtube.com/playlist?list=PLTydW-y9HsbQ2ztoaLBJTd4wwjc_oqWx4) - Training with Live Project
+* [mysql bangla tutorial | Mysql Database | HSC,CSE, Anyone](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqp4x6XstgFCVi6XVu37t99) - Learn Hunter
```

Comment:
- Needs vertical tab scape. See issue #5176
- Instructor as seen in his profile: `Sohidul Islam` @sohidulIslam353

```suggestion
* [mysql bangla tutorial \| Mysql Database \| HSC,CSE, Anyone](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqp4x6XstgFCVi6XVu37t99) - Sohidul Islam, Learn Hunter (YouTube)
```